site/pt/_partials/devtools/whats-new.md

```diff
@@ -4,19 +4,17 @@ Consulte a versão em inglês do <a href="/tags/new-in-devtools/" translate="no"
 
 {# $content #}
 
-{# ### Chrome 110 {: #chrome110 }
-
-* [Clearing Performance Panel on reload](/pt/blog/new-in-devtools-110/#perf)
-* [Recorder updates](/pt/blog/new-in-devtools-110/#recorder)
-* [View and highlight the code of your user flow in the Recorder](/pt/blog/new-in-devtools-110/#recorder-code)
-* [Customize selector types of a recording](/pt/blog/new-in-devtools-110/#recorder-selector)
-* [Edit user flow while recording](/pt/blog/new-in-devtools-110/#recorder-edit)
-* [Automatic in-place pretty print](/pt/blog/new-in-devtools-110/#pretty-print)
-* [Better syntax highlight and inline preview for Vue, SCSS and more](/pt/blog/new-in-devtools-110/#highlight)
-* [Ergonomic and consistent Autocomplete in the Console](/pt/blog/new-in-devtools-110/#console)
-* [Miscellaneous highlights](/pt/blog/new-in-devtools-110/#misc)
- #}
-
+### Chrome 110 {: #chrome110 }
+
+* [Limpeza do painel Desempenho ao recarregar](/pt/blog/new-in-devtools-110/#perf)
+* [Atualizações do gravador](/pt/blog/new-in-devtools-110/#recorder)
```

Comment:
Gravador

(be consistent with the wording below which capitalizes the first letter)

Reply:
Done